lib/ae_mdw/util.ex

```diff
@@ -188,22 +190,26 @@ defmodule AeMdw.Util do
     )
   end
 
-  @spec max_256bit_int() :: integer()
-  def max_256bit_int(),
-    do: 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
+  @spec max_int() :: nil
+  def max_int(), do: nil
 
   @spec max_256bit_bin() :: binary()
-  def max_256bit_bin(), do: <<max_256bit_int()::256>>
-
-  @spec max_name_bin() :: binary
-  def max_name_bin(), do: String.duplicate("z", 128)
+  def max_256bit_bin(),
+    do: <<0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF::256>>
 
-  @spec min_int() :: integer()
-  def min_int(), do: -100
+  @spec max_name_bin() :: binary()
+  def max_name_bin(), do: @max_name_bin
 
   @spec min_bin() :: binary()
   def min_bin(), do: <<>>
 
+  # minimum small integer from https://www.erlang.org/doc/efficiency_guide/advanced.html
+  @spec min_int() :: integer()
+  def min_int(), do: -576_460_752_303_423_488
+
+  @spec min_256bit_int() :: integer()
+  def min_256bit_int(), do: -0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
```

Comment:
Not sure why we're using either `min_int()` or `min_256bit_int()`, but we should probably always use just one, I can't find a reason why we wouldn't

Reply:
`min_256bit_int` was left for one case that needs to be investigated when NFT token id is negative.